sympy/stats/matrix_distributions.py

```diff
@@ -151,3 +151,148 @@ def MatrixGamma(symbol, alpha, beta, scale_matrix):
     if isinstance(scale_matrix, list):
         scale_matrix = ImmutableMatrix(scale_matrix)
     return rv(symbol, MatrixGammaDistribution, (alpha, beta, scale_matrix))
+
+#-------------------------------------------------------------------------------
+# Wishart Distribution ----------------------------------------------------------
+
+def Wishart(symbol, n, scale_matrix):
+    """
+    Creates a random variable with Wishart Distribution.
+
+    The density of the said distribution can be found at [1].
+
+    Parameters
+    ==========
+
+    n: Positive Real number
+        Represents degrees of freedom
+    scale_matrix: Positive definite real square matrix
+        Scale Matrix
+
+    Returns
+    =======
+
+    RandomSymbol
+
+    Examples
+    ========
+
+    >>> from sympy.stats import density, Wishart
+    >>> from sympy import MatrixSymbol, symbols
+    >>> n = symbols('n', positive=True)
+    >>> W = Wishart('W', n, [[2, 1], [1, 2]])
+    >>> X = MatrixSymbol('X', 2, 2)
+    >>> density(W)(X).doit()
+    2**(-n)*3**(-n/2)*exp(Trace(Matrix([
+    [-1/3,  1/6],
+    [ 1/6, -1/3]])*X))*Determinant(X)**(n/2 - 3/2)/(sqrt(pi)*gamma(n/2)*gamma(n/2 - 1/2))
+    >>> density(W)([[1, 0], [0, 1]]).doit()
+    2**(-n)*3**(-n/2)*exp(-2/3)/(sqrt(pi)*gamma(n/2)*gamma(n/2 - 1/2))
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Wishart_distribution
+
+    """
+    if isinstance(scale_matrix, list):
+        scale_matrix = ImmutableMatrix(scale_matrix)
+    return rv(symbol, MatrixGammaDistribution, (n/S(2), S(2), scale_matrix))
```

Comment:
Make a new class for encapsulating common features of `MatrixGammaDistribution` and `WishartDistribution`, then inherit it in both these classes. Reuse the code from this class. Something like,
```python
class _HelperMatrixGammaWishart(MatrixDistribution):
    
    _argnames = ('scale_matrix')
    
    @staticmethod
    def check(scale_matrix):
        ...

    @property
    def set(self):
        ...

    @property
    def dimension(self):
        ...

    def pdf(self, x, alpha, beta):
        ...
```